### PR TITLE
Add reader for VIIRS Active Fires

### PR DESCRIPTION
**Pull request by rdaruwala**
Adds readers for the VIIRS Active Fires data format. Reads NetCDF4 and CSV (.txt) files

 - [x] Tests added 
 - [x] Tests passed 
 - [x] Passes ``git diff origin/master -- "*py" | flake8 --diff``
 - [x] Fully documented

### CODE REVIEW

File: satpy/tests/reader_tests/test_viirs_fires.py [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+        25.90660477, -100.06127167,  331.17962646,   0.75,   0.75,   81,   20.61096764''')
+
+        return dd.from_pandas(pd.read_csv(fake_file, skiprows=15, header=None,
+                                          names=["latitude", "longitude",
+                                                 "T13", "Along-scan", "Along-track",
+                                                 "detection_confidence",
+                                                 "power"]), chunksize=1)
+
+
+class TestVIIRSACTIVEFIRES_CDF4(unittest.TestCase):
```
Comment (djhoese) [inline satpy/tests/reader_tests/test_viirs_fires.py]: Please make these PEP8 names: `TestVIIRSActiveFiresNetCDF4` or something like that. `VIIRS` is an acronym so it should be all caps. Other class name needs to be updated too.

File: satpy/readers/viirs_active_fires.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
+from satpy.readers.netcdf_utils import NetCDF4FileHandler
+from satpy.readers.file_handlers import BaseFileHandler
+import os
+import dask.dataframe as dd
+import xarray as xr
+
+
+class VIIRSActiveFiresFileHandler(NetCDF4FileHandler):
+
+    def get_dataset(self, dsid, dsinfo):
```
Comment (djhoese) [inline satpy/readers/viirs_active_fires.py]: Docstrings for the class, this method, and maybe the properties would be nice. The `get_dataset` should include google style `Args:` documentation.

File: satpy/readers/viirs_active_fires.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
+from satpy.readers.netcdf_utils import NetCDF4FileHandler
```
Comment (djhoese) [inline satpy/readers/viirs_active_fires.py]: This module and all python modules you've created should include the header like the rest of satpy. The copyright author should be "SatPy developers"...I think (I keep changing my mind, but I think this is what I'm going with now). Make sure blank lines still have a `#`. The module should have a docstring `"""` too.

File: satpy/etc/readers/viirs_active_fires.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
+reader:
+    description: VIIRS Active Fires Reader
+    name: viirs_active_fires
```
Comment (djhoese) [inline satpy/etc/readers/viirs_active_fires.yaml]: I think the name of the reader (and the name of this YAML file) should be `viirs_edr_active_fires`. Maybe `viirs_edr_fires` (@adybbroe @martin thoughts?).

Comment (rdaruwala) [reply satpy/etc/readers/viirs_active_fires.yaml]: I chose`viirs_edr_fires`. Let me know if you'd prefer `viirs_edr_active_fires`.

File: satpy/readers/viirs_edr_active_fires.py [hunk 5]
```diff
@@ -0,0 +1,114 @@
+# Copyright (c) 2019 SatPy Developers
+#
+# Author(s):
+#
+#   Rohan Daruwala <rdaruwala@wisc.edu>
```
Comment (djhoese) [inline satpy/readers/viirs_edr_active_fires.py]: Please remove the author section here and instead add yourself to the AUTHORS file in the root of the repository. I'm trying to get away from individual authors in each module. They become out of date and not representative of the real authors.

Comment (rdaruwala) [reply satpy/readers/viirs_edr_active_fires.py]: Done!

**Comment by mraspaud** [comment]
Merged, thanks for your contribution!

**Pull request merged**